cfgmgr/teammgr.cpp

```diff
@@ -112,18 +112,53 @@ void TeamMgr::doTask(Consumer &consumer)
     }
 }
 
-
 void TeamMgr::cleanTeamProcesses()
 {
     SWSS_LOG_ENTER();
     SWSS_LOG_NOTICE("Cleaning up LAGs during shutdown...");
-    for (const auto& it: m_lagList)
+
+    std::unordered_map<std::string, pid_t> aliasPidMap;
+
+    for (const auto& alias: m_lagList)
+    {
+        std::string res;
+        pid_t pid;
+
+        {
+            std::stringstream cmd;
+            cmd << "cat " << shellquote("/var/run/teamd/" + alias + ".pid");
+            EXEC_WITH_ERROR_THROW(cmd.str(), res);
+
+            pid = static_cast<pid_t>(std::stoul(res, nullptr, 10));
+            aliasPidMap[alias] = pid;
+
+            SWSS_LOG_INFO("Read port channel %s pid %d", alias.c_str(), pid);
+        }
+
+        {
+            std::stringstream cmd;
+            cmd << "kill -TERM " << pid;
+            EXEC_WITH_ERROR_THROW(cmd.str(), res);
```

Comment:
Although the performance is sufficient, we can probably gain more by replacing this part with signals API